### ISSUE TITLE
Accordion-item should not have role="region"

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#594]
RE: https://modus-web-components.trimble.com/?path=/story/components-accordion--default

![image](https://user-images.githubusercontent.com/1212885/180993369-dddc8a24-1527-4199-ba55-03a6c0a57f5c.png)

There are multiple accordion-items so this fails a11y tests.
`role="region"` isn't needed for accordion-items at all anyway so we should remove.
